migrations/20240925112800_create_search_message_table.up.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS messages (
+    id                UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+
+    team_id           UUID NOT NULL,
+    message_id        UUID NOT NULL UNIQUE,
+
+    content           tsvector,
+    /* This field content both the target name and the public identifier */
+    target_name       tsvector,
+
+    updated_at        TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW()
+);
+
+--bun:split
+
+CREATE INDEX IF NOT EXISTS message_search ON messages USING GIN ( (messages.content || messages.target_name) );
```

Comment:
This index should also include `team_id`, refer to the previous message for the steps to do.